### PR TITLE
Tests on content script

### PR DESCRIPTION
**Pull request by river-honer**
`styleTweet.test.js`
* Tests that the styles are applied to tweets or not applied to tweets according to the input and the options.

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ globals:
 parserOptions:
   ecmaVersion: 2018
   sourceType: module
+jest: true
```
Comment (ribicn-softerrific) [inline .eslintrc.yml]: No need for this, #98 adds config for tests for extension

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "start:firefox": "concurrently 'npm run webpack:dev -- --watch' 'web-ext run --config=.web-ext-config.js'",
     "build:prod": "npm run webpack:prod && web-ext build --config=.web-ext-config.js",
     "test": "jest --coverage && npm run webpack:prod && web-ext lint -s dist",
+    "test:local": "jest",
```
Comment (ribicn-softerrific) [inline package.json]: Also, no need for this.

Comment (river-honer) [reply package.json]: it looks like our other test script is used by ci or something, it's way too much for just running tests.

Comment (river-honer) [reply package.json]: i would keep this

Comment (ribicn-softerrific) [reply package.json]: Yes, it's ok.

**Pull request merged**